### ISSUE TITLE
A11y_AzureToolsStorageExplorer_SigningintoAzureStorage_ToggleExplorerPaneTreeView_Zoom: At 400% zoom controls(Open Connect Dialog, Settings, Send Feedback) present on left pane gets overlapped and controls present under Treeview screen are not visible.

### ISSUE THREAD

**Issue by shreyashi227** [microsoft/AzureStorageExplorer#2618]
**GitHub Tags**: #A11y_AzureToolsStorageExplorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp; #Win32; #Benchmark; #A11ySev2; #WCAG1.4.10; #Zoom; #Azure Tools Storage Explorer;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.12.0
Operating System :Windows10

**Prerequisite:**
1) Set the display resolution of system at 1280x1024.
2) Click on "Advanced scaling settings"
3) Set custom scaling size at 400% and click on "Apply" 

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Traverse through the treeview opened on left of screen for whatever tasks or subscription selected.
5. Check if at 400% zoom controls(Open Connect Dialog, Settings, Send Feedback) present on left pane gets overlapped and controls present under Treeview screen are not visible.

**Actual:**
At 400% zoom controls present on left pane gets overlapped and controls present under Treeview screen are not visible.

**Same issue is observed in whole application.**

**Expected**:
Every content on screen should be clearly visible and accessible even at 400% zoom 

**User Impact:**
Low vision users will face difficulty in accessing and clearly viewing the contents present on screen.

- [x] Prevent left vertical toolbar buttons from overlapping, which affect clicking.
- [x] Prevent treeview UI and properties/actions UI from overlapping, which affect reading/clicking. (I cannot reproduce this anymore, not sure why)
- [x] Make the scrollable section of Properties panel cover not only keys but also values.
- [x] Make explorer section more space efficient so the table section is visible when zoom level is high.

**Comment by shreyashi227** [comment]
We have uploaded repro screenshot [OneDrive ](https://microsoft-my.sharepoint.com/personal/v-subha_microsoft_com/_layouts/15/onedrive.aspx?originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOmY6L3Avdi1zdWJoYS9FdnhIZGcxemtOOVBrbkYxdU94bk9pUUJlanB4S1lnOW4yMUR5ZGdRX29tMGV3P3J0aW1lPWJNTVh6dTJ1MTBn&id=%2Fpersonal%2Fv%2Dsubha%5Fmicrosoft%5Fcom%2FDocuments%2FMicrosoft%20Azure%20Tools%20Storage%20Explorer%20Attachments%2F2618%5FA11y%5FAzureToolsStorageExplorer%5FSigningintoAzureStorage%5FToggleExplorerPaneTreeView%5FZoom%2Epng&parent=%2Fpersonal%2Fv%2Dsubha%5Fmicrosoft%5Fcom%2FDocuments%2FMicrosoft%20Azure%20Tools%20Storage%20Explorer%20Attachments)link with name "2618_A11y_AzureToolsStorageExplorer_SigningintoAzureStorage_ToggleExplorerPaneTreeView_Zoom" please refer the Screenshot for reference.
Please let us know if there is any problem in accessing the link

**Comment by JasonYeMSFT** [comment]
Let's study the spec to see if this rule is applicable to us: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BE12828DF-B35C-4F4D-9289-1F932E19F8CF%7D&file=MAS&cid=bc1f22c1-53e5-4adf-bad5-970610342251

**Comment by JasonYeMSFT** [comment]
Moving out of 1.16.0 since WCAG bugs have a later deadline.

**Comment by JasonYeMSFT** [comment]
Had a discussion with accessibility experts, we have at least the following things to be done.
- [ ] Prevent left vertical toolbar buttons from overlapping, which affect clicking.
- [ ] Prevent treeview UI and properties/actions UI from overlapping, which affect reading/clicking.
- [ ] Make the scrollable section of Properties panel cover not only keys but also values.
- [ ] Make explorer section more space efficient so the table section is visible when zoom level is high.

**Comment by shreyashi227** [comment]
@JasonYeMSFT

Verified the issue at version 1.19.
Issue is still reproducing and controls present under Treeview screen are not visible.

**Comment by JasonYeMSFT** [comment]
@shreyashi227 Have you tried tuning Storage Explorer's zoom level (zoom out) to cancel the effect of the system zoom level? During my last consulting with the accessibility engineers, I was told that we need to be accessible under such zoom level, not showing everything in exactly 400% size but still be accessible. That is to say, the user can tune the in-app zoom level, collapse/expand/resize section of the UI to access the UI elements.

This is what I can achieve with 400% system zoom level if I zoom out a little bit in Storage Explorer.
![400-zoom](https://user-images.githubusercontent.com/39359541/119020886-9c431c00-b953-11eb-9f7d-648e590db472.JPG)

**Comment by shreyashi227** [comment]
Verified the issue at version 1.19 and using the above method of Storage Explorer's zoom level (zoom out) to cancel the effect of the system zoom level. And now the content is visible properly.

**Comment by fsteffi** [comment]
**GitHub Tags:**#A11yMAS;#BM-Azure Tools Storage Explorer-win32-Aug2020;

**Comment by 12-shweta** [comment]
Github Tags: #Closed;#Regressed:02-03-23
